### PR TITLE
Hossein changes related record #1037

### PR DESCRIPTION
**Pull request by hoseinmirian**
Please check array express record

image

### CODE REVIEW

File: src/components/Records/Record/Collections.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
     tile
     elevation="3"
   >
-    <SectionTitle title="Collections" />
+    <SectionTitle title="In colledted" />
```
Comment (knirirr) [inline src/components/Records/Record/Collections.vue]:
```suggestion
    <SectionTitle title="Collections &amp; Recommendations" />
```

**Review by knirirr — CHANGES_REQUESTED** [review]
There's a discussion of this change here:

https://pyroeis.oerc.ox.ac.uk/#narrow/stream/8-FAIRsharing-Rebuild/topic/Front-end/near/30004

One of the records mentioned is https://deploy-preview-1045--fairsharing.netlify.app/#/FAIRsharing.tsp58, which should collect at least one record (https://deploy-preview-1045--fairsharing.netlify.app/#/FAIRsharing.q81k8w). But, there's no related records box at all.

File: src/components/Records/Record/RelatedContent.vue [hunk 2]
```diff
@@ -158,7 +158,7 @@ export default {
           }
           else {
             _module.tabsData.tabs[tabName].data = _module.prepareAssociations(_module.currentRecord['fairsharingRecord'].recordAssociations, _module.currentRecord['fairsharingRecord'].reverseRecordAssociations)
-                .filter(item => _module.tabsData.tabs[tabName].registry.includes(item.registry) && item.linkType !=='fairsharingRecord' && !_module.tabsData.tabs[tabName].relation.includes(item.recordAssocLabel) )
+                .filter(item => _module.tabsData.tabs[tabName].registry.includes(item.registry) && item.linkType !=='fairsharingRecord' && _module.tabsData.tabs[tabName].relation.includes(item.recordAssocLabel) )
```
Comment (knirirr) [inline src/components/Records/Record/RelatedContent.vue]: The removal of this `!` has caused the breakage mentioned in the comments (disappearing related records).

**Comment by knirirr** [comment]
There's one other change still needed - that is, the related records box should show outgoing recommends relationships. It appears not to do so at present (compare the two records linked above). 

It might be worth doing this by refactoring prepareAssociations as I have commented elsewhere.

**Comment by hoseinmirian** [comment]
please test record:
http://localhost:8080/#/FAIRsharing.dIDAzV

**Review by knirirr — APPROVED** [review]

**Pull request merged**